app/models/academic_year.rb

```diff
@@ -1,6 +1,9 @@
 class AcademicYear < ApplicationRecord
   has_many :placement_preferences
 
+  attribute :starts_on, :date
+  attribute :ends_on, :date
```

Comment:
I've added this temporarily to fix the issues caused by starts_on and ends_on being converted into strings, any idea on why this has happened would be much appriciated.